Dockerfile

```diff
@@ -1,13 +1,25 @@
 FROM alpine:3.14 AS builder
 
 RUN apk add --no-cache tini gcc g++ pkgconf make \
-	util-linux-dev sqlite-dev mariadb-connector-c-dev cmake zlib-dev fmt-dev \
-	file-dev libexif-dev curl-dev ffmpeg-dev ffmpegthumbnailer-dev \
-	libmatroska-dev libebml-dev taglib-dev pugixml-dev spdlog-dev \
-	duktape-dev libupnp-dev git bash
+    util-linux-dev sqlite-dev mariadb-connector-c-dev cmake zlib-dev fmt-dev \
+    file-dev libexif-dev curl-dev ffmpeg-dev ffmpegthumbnailer-dev \
+    libmatroska-dev libebml-dev taglib-dev pugixml-dev spdlog-dev \
+    duktape-dev git bash
 
-WORKDIR /gerbera_build
+# Build libupnp
+WORKDIR /libupnp_build
+RUN wget -c https://github.com/pupnp/pupnp/archive/refs/tags/release-1.14.12.tar.gz -O - | tar -xz && \
```

Comment:
There is a script install-pupnp.sh. Can you try to integrate that here. Get the right version and settings.

Reply:
done